build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.1'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```

Comment:
@paddybyers but isn't this change specifically to address the requirement of using the version from `BuildConfig`, see https://github.com/ably/ably-java/pull/143/commits/10e3948d290d71b6af9add400e0f59c01a6d1a61#diff-5d450029acab7ff9e9ffca5408321058L20?